pytensor/link/c/op.py

```diff
@@ -334,12 +338,11 @@ def __init__(self, func_files: str | list[str], func_name: str | None = None):
                     "Cannot have an `op_code_cleanup` section and specify `func_name`"
                 )
 
-    def load_c_code(self, func_files: list[str]) -> None:
+    def load_c_code(self, func_files: list[Path]) -> None:
```

Comment:
```suggestion
    def load_c_code(self, func_files: Iterable[Path]) -> None:
```

Reply:
This broke the `func_files[i]` in the error messages but I removed them so it's fine now.